client/ayon_core/hosts/houdini/plugins/publish/validate_mesh_is_static.py

```diff
@@ -16,9 +16,13 @@ class ValidateMeshIsStatic(pyblish.api.InstancePlugin,
     """Validate mesh is static.
 
     It checks if output node is time dependent.
+    this avoids getting different output from ROP node when extracted
+    from a different frame than the first frame.
+    (Might be overly restrictive though)
     """
 
-    families = ["staticMesh"]
+    families = ["staticMesh",
```

Comment:
what's the difference between `staticMesh` and `model`?

Reply:
I believe staticMesh is unreal-specific- but I might be mistaken :/

Reply:
[staticMesh](https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/hosts/houdini/plugins/create/create_staticmesh.py) in Houdini is similar to [staticMesh](https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/hosts/maya/plugins/create/create_unreal_staticmesh.py) in maya. 

More info about it here: 
https://ayon.ynput.io/docs/addon_houdini_artist#static-mesh-fbx